docs/recipes/installTips.md

```diff
@@ -364,6 +363,254 @@ GUI will open!
+
+Example: the RTX 3080 has a nvidia driver version (525.60) that is unsupported by CUDA 11. But nvidia version 510 is. Therefore this is sufficient:
+
+`sudo apt install nvidia-driver-510`
+
+then:
+
+`reboot -n`
+
+re-open terminal and check gcc version:
+
+`gcc --version`
+
+output:
+```python
+gcc --version
+gcc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0
+Copyright (C) 2019 Free Software Foundation, Inc.
+This is free software; see the source for copying conditions.  There is NO
+warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+```
+Then finish installation:
+
+`sudo apt install nvidia-cuda-toolkit gcc-9`
+
+Then check:
+
+`nvcc --version`
+
+If error messages, read them carefully as they often tell you how to fix it, or what to google :D
+
+Now you can see CUDA, DRIVER, GPU(s):
+
+`nvidia-smi`
+
+output:
+
+```python
+$ nvidia-smi
+Sun Jan  1 10:14:06 2023       
++-----------------------------------------------------------------------------+
+| NVIDIA-SMI 510.108.03   Driver Version: 510.108.03   CUDA Version: 11.6     |
+|-------------------------------+----------------------+----------------------+
+| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
+| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
+|                               |                      |               MIG M. |
+|===============================+======================+======================|
+|   0  NVIDIA GeForce ...  Off  | 00000000:01:00.0 Off |                  N/A |
+|  0%   49C    P8    17W / 350W |      5MiB / 24576MiB |      0%      Default |
+|                               |                      |                  N/A |
++-------------------------------+----------------------+----------------------+
+                                                                               
++-----------------------------------------------------------------------------+
+| Processes:                                                                  |
+|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
+|        ID   ID                                                   Usage      |
+|=============================================================================|
+|    0   N/A  N/A      1275      G   /usr/lib/xorg/Xorg                  4MiB |
++-----------------------------------------------------------------------------+
+
+```
+
+Sweet!
+
+
+### Next, Anaconda!
+
+Click here to get the ubuntu/linux package: https://www.anaconda.com/products/individual#linux
+
+this downloads a file, save it somewhere. (~/Downloads is OK)
+
+then enter the folder:
+
+`cd Downloads`
+
+and run:
+
+`bash Anaconda3-2021.05-Linux-x86_64.sh`
```

Comment:
Here we could be a bit more explicit that the filename should be updated depending on the file downloaded.

We could also get users to install `miniconda` instead of Anaconda, which would be a bit more lightweight. Users should be able to do this entirely from the command line (haven't tested this installation exactly, as I do something similar but when installing miniconda in docker containers):

Note that depending on their system architecture, different versions should be installed (e.g., could be `-aarch64.sh` instead of `-x86.sh`)

```
wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86.sh -O miniconda.sh -q
bash miniconda.sh -b
```